src/neo-vm/Types/Null.cs

```diff
@@ -21,14 +21,24 @@ public override bool Equals(StackItem other)
             return other is Null;
         }
 
+        public override bool GetBoolean()
+        {
+            return false;
+        }
+
         public override int GetHashCode()
         {
             return 0;
         }
 
-        public override bool ToBoolean()
+        public override T GetInterface<T>()
         {
-            return false;
+            return null;
```

Comment:
throw exception?

Reply:
The object `StackItem.Null` could be any type. This means that the contract want to send a `null` to us.